plutus-use-cases/test/Spec/crowdfundingTestOutput.txt

```diff
@@ -21,25 +21,25 @@ Events by wallet:
     - Iteration: 3
     Requests:
         4: {utxo-at:
-            ScriptAddress: d0b2549a3cff9ef9e2b93b273a84ff7cb08a06eebe536f9ca3279825dde20e4f}
```

Comment:
why has this stuff changed?

Reply:
I think it's just non-deterministic? @michaelpj 

By the way, the CBOR encoding has changed. Is there anything that needs to be updated?

Reply:
Script addresses are hashes of serialized scripts. The way we encode builtins has changed, therefor the script hashes have changed.